pkg/pillar/cmd/msrv/handlers.go

```diff
@@ -524,13 +524,55 @@ func HandleAppCustomBlobs(z *Msrv) func(http.ResponseWriter, *http.Request) {
 	}
 }
 
+func HandleAppInstanceDiscovery(srv *Msrv) func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		remoteIP := net.ParseIP(strings.Split(r.RemoteAddr, ":")[0])
+		appStatus := srv.lookupAppInstStatusByAppIP(remoteIP)
+		if appStatus == nil {
+			msg := fmt.Sprintf("No AppNetworkStatus for %s", remoteIP.String())
+			srv.Log.Errorf(fmt.Sprintf("HandleAppInstanceDiscovery: %s", msg))
```

Comment:
`Errorf` does not take a format string?

Reply:
oh, yes, you're right, Sprintf is not needed here :)